compiler/test/compilable/test23235.d

```diff
@@ -0,0 +1,20 @@
+/* https://issues.dlang.org/show_bug.cgi?id=23235
+ */
+
+@safe:
```

Comment:
should this test case be setting `-preview=dip1000`?

Reply:
It works either way.